data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Original:</p><h2>Copy prices for the week </h2><p> </p><p>The Citibike dataset contains values for each day, however, the Gas Prices dataset contains one value per week. To get values for each day, you can copy the price of the week's value for the entire seven days. </p><p> </p><p>Proposed:</p><h2>Fill missing price data</h2><p> </p><p>The Citibike dataset contains <strong>pricing</strong> values for each day, however, the Gas Prices dataset contains one value per week. To get <strong>daily pricing</strong> values, you can <strong>assign</strong> <strong>a week's pricing value to</strong> <strong>each day of that particular week</strong>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Done</p>